js/extension.js

```diff
@@ -108,15 +111,16 @@ function loadTasks(context) {
             if (getValue2(task, config, "options", "statusbar") == "hide") {
                 hide[computeId(task, config)] = true;
             }
+            statusbarLabels[computeId(task, config)] = getValue2(task, config, "options", "statusbarLabel");
```

Comment:
add an item "statusbarLabel" to "options" of tasks.json,if it's configed,this extentsion use it as the task name show on statusbar